.github/workflows/package-Exporter.Stackdriver.yml

```diff
@@ -0,0 +1,50 @@
+name: Pack OpenTelemetry.Contrib.Exporter.Stackdriver
```

Comment:
todo: add a section in contributing.md about new projects, that it require minver prefix and a dedicated workflow.

Reply:
Can we get this one merged? And I'll do a separate PR for contribution guidance.